include/FreeRTOSConfig.h

```diff
@@ -105,7 +105,7 @@
 #define configMAX_TASK_NAME_LEN		  ( 16 )
 #define configIDLE_SHOULD_YIELD		  1
 #define configGENERATE_RUN_TIME_STATS 1
-#define configUSE_MALLOC_FAILED_HOOK  1
+#define configUSE_MALLOC_FAILED_HOOK  0

```

Comment:
Not sure what this does, but it reminds me there were times we wanted to add a log message or traceback (from SEGVs) for postmortem analysis.

Reply:
This is different from that IMO. Malloc's should just be checked for NULL and the satellite continues on. We can always reboot it